docs/permissions/data-sandboxes-draft.md

```diff
@@ -0,0 +1,309 @@
+
+For an example, see [Advanced data sandboxing: limiting access to columns](https://www.metabase.com/learn/permissions/data-sandboxing-column-permissions).
+
+## Restricting rows in an advanced sandbox with user attributes
+
+You can set up an advanced sandbox to display different rows to each person depending on their [user attributes](../people-and-groups/managing.md#adding-a-user-attribute). For example, you can display the "Sandboxed Accounts" question with the filter `Plan = "Basic"` for one group, and the filter `Plan = "Premium"` for another group.
+
+1. Make sure you've done all the [prerequisites for advanced sandboxes](#prerequisites-for-advanced-sandboxes).
+2. Go to the SQL question that will be displayed to the people in the advanced sandbox.
+3. Add a [parameterized](../questions/native-editor/sql-parameters.md) `WHERE` clause to your SQL query, such as `{%raw%}[[ WHERE plan = {{ plan_variable }} ]]{%endraw%}`.
+4. Save the SQL question.
+5. Go to **Admin settings** > **Permissions**.
+6. Find the group and table for your advanced sandbox.
+7. Open the dropdown under **Data access**.
+8. Click **Edit sandboxed access**.
+9. Scroll down and set **Parameter or variable** to the name of the parameter in your saved SQL question (such as "Plan Variable").
+10. Set the **User attribute** to the **key** of the [user attribute](../people-and-groups/managing.md#adding-a-user-attribute) to be filtered on (such as "User's Plan").
+11. Click **Save**.
+
+### How row restriction works in an advanced sandbox
+
+How user attributes, SQL parameters, and advanced sandbox permissions work together to display different rows to different people.
+
+A standard `WHERE` clause filters a table by setting a column to a fixed value:
+```
+WHERE column_name = column_value
+```
+
+In step 2 of the [row restricting configuration](#restricting-rows-in-an-advanced-sandbox-with-user-attributes) above, you'll add a SQL variable so that the `WHERE` clause will accept a dynamic value:
+```
+WHERE plan = {%raw%}{{ plan_variable }}{%endraw%} 
+```
+
+In steps 9-10 of the [row restricting configuration](#restricting-rows-in-an-advanced-sandbox-with-user-attributes) above, you're telling Metabase to map the SQL variable `plan_variable` to a **user attribute key** (such as "User's Plan"):
+```
+WHERE plan = USER_ATTRIBUTE_KEY
+```
+
+Metabase will use the key to look up the **user attribute value** (for example, the "User's Plan" key can map to the values "Basic", "Business", or "Premium"):
+```
+WHERE plan = USER_ATTRIBUTE_VALUE
+```
+
+Metabase replaces the SQL parameter with the specific **user attribute value** (such as "Basic") associated with a sandboxed person. When that person logs into Metabase and uses the sandboxed table, they'll see the query result that uses:
+```
+WHERE plan = "Basic"
+```
+
+## How sandboxing permissions interact with other permissions
```

Comment:
Now I'm thinking we're oversharing here, as in giving people way too much to read that they should haven't to.

The important info here is where sandboxing rules break down, or get iffy.

Everything you have in with "uses sandboxing rules" set to "always" we can just gloss over. In fact, we shouldn't mention it. We can say something like "Sandboxing rules apply everywhere in Metabase for a group, with the following caveats:" And then just stick to the edge cases that people might need to consider. We shouldn't, for example, make them read a section on the Data Browser to figure out that, yes, looks like we're all good there.

Reply:
Good point! I wrote it out like this because if I was the person configuring permissions for sensitive data, I would want to check each place where the sandboxed table appears (just to make sure I did the permissions correctly).

How about we keep a comprehensive list in the table (so people can verify if they want), but eliminating the detailed sections for the "Always" line items?